pieceOfCake5_2.py

```diff
@@ -0,0 +1,10 @@
+def get_recipe_price(prices,optionals=None,**ingredients):
+  pricesCopy=copy.deepcopy(prices)
```

Comment:
Is there a way to ignore the ingridients in optionals without deep copying prices?